cardano-db/src/Cardano/Db/Migration/Extra/CosnumedTxOut/Queries.hs

```diff
@@ -202,40 +206,70 @@ countConsumed = do
     pure countRows
   pure $ maybe 0 unValue (listToMaybe res)
 
+deleteAndUpdateConsumedTxOut :: MonadIO m => Trace IO Text -> Word64 -> ReaderT SqlBackend m ()
+deleteAndUpdateConsumedTxOut trce blockNoDiff = do
+  maxTxInId <- findMaxTxInId blockNoDiff
+  case maxTxInId of
+    Left errMsg -> liftIO $ logInfo trce $ "No tx_out were deleted: " <> errMsg
```

Comment:
I think this should be handled differently. If we don't find a `maxTxInId` this means there are no values to delete. However, there are still values to update and the extra column should still be created.

Reply:
yeah think you are right, not sure if it's related but the existing prune/consume tests are failing because they always expect `migrateTxOut` to be ran.
I'm actually not quite grasping it tbh as I don't fully understand the functionality and order of how we handle UTXOs from reading the code :)